charts/revere/values.yaml

```diff
@@ -0,0 +1,211 @@
+# -- (object) Required; contents of revere.yaml to be given to the application
+config:
+
+# Global values used by this chart
+global:
+  # -- (string) What version of the application to deploy
+  applicationVersion: latest
+
+# name -- A name for the deployment that will be substituted into resource definitions
+name: revere
+
+imageConfig:
+  # -- (string) Image repository
+  repository: us-central1-docker.pkg.dev/dsp-artifact-registry/revere/revere
+  # -- (string) Image tag
+  # @default -- global.applicationVersion
+  tag: null
+  # -- (string) When to pull images
+  imagePullPolicy: Always
+
+# -- (number) Number of replicas for the deployment
+replicas: 3
+
+# -- (map) Annotations for application pods
+annotations: {}
+
+# -- (string) Optional override for the pod's runtimeClass
+runtimeClassName:
+
+resources:
+  requests:
+    # -- (string) Number of CPU units to request for the deployment
+    cpu: 500m
+    # -- (string) Memory to request for the deployment
+    memory: 4Gi
+  limits:
+    # -- (string) Number of CPU units to limit the deployment to
+    cpu: 500m
+    # -- (string) Memory to limit the deployment to
+    memory: 4Gi
+
+probes:
+  readiness:
```

Comment:
Do you plan on adding any configurations to these probes? I think these should mirror how [we define probes in other charts](https://github.com/broadinstitute/terra-helm/blob/d1ee696637052dc2c35ef3b06e21f2974ec7e935/charts/buffer/values.yaml#L39)s.

Reply:
At some point, definitely! Revere doesn't have any ingress at all right now--it literally doesn't run a server at all, it is just a client of Statuspage.io and Google PubSub. Once it runs a server or something we'll have to come through and add that capability to the chart anyway, I guess I just wanted these placeholders here so I could add probes for testing purposes.